### PR TITLE
Add support for uptimerobot public status pages

### PR DESCRIPTION
**Pull request by daveshepherd**

### CODE REVIEW

**Comment by daveshepherd** [comment]
Feel free to give me feedback on this.

**Comment by stakater-user** [comment]
@daveshepherd Image is available for testing. ``docker pull stakater/ingressmonitorcontroller:PR-136-1``

**Comment by rasheedamir** [comment]
great @daveshepherd ! @waseem-h will review it tomorrow

File: pkg/monitors/uptimerobot/uptime-status-page_test.go [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+	}
+	if ! util.ContainsString(statusPageIds, statusPage2.ID) {
+		t.Error("The second status page does not contain the monitor, expected: " + statusPage2.ID + ", but was: " + strings.Join(statusPageIds, "-"))
+	}
+
+	if util.ContainsString(statusPageIds, statusPage3.ID) {
+		t.Error("The third status page should not contain the monitor, but was: " + strings.Join(statusPageIds, "-"))
+	}
+
+    // Tidy up
```
Comment (waseem-h) [inline pkg/monitors/uptimerobot/uptime-status-page_test.go]: Please fix indentations and stuff

File: pkg/monitors/uptimerobot/uptime-responses.go [hunk 2]
```diff
@@ -26,8 +26,6 @@ type UptimeMonitorMonitor struct {
 	Interval       int                 `json:"interval"`
 	Status         int                 `json:"status"`
 	CreateDatetime int                 `json:"create_datetime"`
-	MonitorGroup   int                 `json:"monitor_group"`
```
[the remark this answers is not in the document]
Comment (daveshepherd) [reply pkg/monitors/uptimerobot/uptime-responses.go]: No, they're not in the api documentation and I've not seen them in the response when querying the API.

**Comment by daveshepherd** [comment]
@waseem-h - I've addressed your comments, let me know if you want me to do anything else.

**Comment by stakater-user** [comment]
@daveshepherd Image is available for testing. ``docker pull stakater/ingressmonitorcontroller:PR-136-2``

**Comment by waseem-h** [comment]
Looks good! @daveshepherd

**Pull request merged**